### PR TITLE
feat(sagas): place sagas into own folders

### PR DESCRIPTION
**Pull request by ChristiaanScheermeijer**
BREAKING CHANGE: existing sagas have to be migrated to new structure.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -152,27 +152,47 @@ module.exports = {
...
-        ]
+        dynamicTask: params => ({
+          task: 'modify',
+          target: 'sagas/index.js',
+          patch: [
+            {
+              pattern: /^import (\w+) from '.\/\w+';\s/gm,
+              custom : function (match, group1) {
+                if (group1 > `${params.name}Sagas` && !params.updatedSagasImport) {
+                  params.updatedSagasImport = true;
```
Comment (ChristiaanScheermeijer) [inline index.js]: not sure about this though...

File: index.js [hunk 2]
```diff
@@ -152,27 +152,47 @@ module.exports = {
...
+                  return `import {{name}}Sagas from './{{pascalCased}}';\n${match}`
+                }
+                return match;
+              }
+            }, {
+              pattern: /^( +)(\w+Sagas)\(\),\n/gm,
+              custom : function (match, group1, group2) {
+                if (group2 > `${params.name}Sagas` && !params.updatedSagasCall) {
+                  params.updatedSagasCall = true;
+                  return `${group1}${params.name}Sagas(),\n${match}`
```
[the remark this answers is not in the document]
Comment (ChristiaanScheermeijer) [reply index.js]: 👍

File: index.js [hunk 3]
```diff
@@ -152,27 +152,47 @@ module.exports = {
...
+                }
+                return match;
```
[the remark this answers is not in the document]
Comment (ChristiaanScheermeijer) [reply index.js]: 👍

File: index.js (hunk 1 above)
Comment (RWOverdijk) [reply index.js]: Ah, well fine then :)

Comment (ChristiaanScheermeijer) [reply index.js]: xD

**Pull request merged**